app/views/layouts/_sort_search_headers.html.haml

```diff
@@ -0,0 +1,16 @@
+- unless defined?(sort_columns)
```

Comment:
thought we were going to name this _sort_headers and then name the old ones _legacy_sort_headers?

Reply:
Just did this now. I updated in the hmpbrands branch as well. Though we will need to update on each project as we go